src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/LanguageServices/WorkspaceContextHost.WorkspaceContextHostInstance.cs

```diff
@@ -52,8 +54,8 @@ public Task InitializeAsync()
 
             protected override async Task InitializeCoreAsync(CancellationToken cancellationToken)
             {
-                _context = await _workspaceProjectContextProvider.Value.CreateProjectContextAsync(_project)
-                                                                       .ConfigureAwait(true);
+                _context = await _workspaceProjectContextProvider.CreateProjectContextAsync(_project)
+                                                                  .ConfigureAwait(true);
```

Comment:
Unrelated to this PR, but do we still need `ConfigureAwait` in some cases?

Reply:
In languageservices - I haven't back ported the rule turn off, so this will break the build if I remove it.